psbt_channel_funder.go

```diff
@@ -110,6 +110,7 @@ func (l *LndPbstChannelFunder) OpenChannel(ctx context.Context,
 			},
 		}),
 		lndclient.WithRemoteReserve(CustomChannelRemoteReserve),
+		lndclient.WithRemoteMaxHtlc(req.RemoteMaxHtlc),
```

Comment:
Should we only set it if the value is above zero? 

Or I guess lnd will just pick that up as not set, and apply the default.

Reply:
I guess that's what would happen, yes. But just to make sure I added a condition.